src/Components/Publishing/Layouts/__tests__/VideoLayout.test.js

```diff
@@ -1,78 +1,86 @@
+import { clone } from "lodash"
 import { mount } from "enzyme"
 import "jest-styled-components"
 import React from "react"
 import renderer from "react-test-renderer"
 import {
   SeriesArticle,
   StandardArticle,
-  VideoArticle
+  VideoArticle,
 } from "../../Fixtures/Articles"
-import { IconVideoPlay } from "../../Icon/IconVideoPlay"
 import { Nav } from "../../Nav/Nav"
-import { ArticleCard } from "../../Series/ArticleCard"
+import { ArticleCard } from "../../RelatedArticles/ArticleCards/ArticleCard"
 import { SeriesAbout } from "../../Series/SeriesAbout"
 import { VideoPlayer } from "../../Video/Player/VideoPlayer"
 import { VideoAbout } from "../../Video/VideoAbout"
 import { VideoLayout } from "../VideoLayout"
+import { ArticleData } from "../../Typings"
 
-jest.mock('../../../../Utils/track.ts', () => ({
-  track: jest.fn()
+jest.mock("../../../../Utils/track.ts", () => ({
+  track: jest.fn(),
 }))
 
 describe("Video Layout", () => {
-  const getWrapper = (props: any = {}) => {
+  const VideoSeriesArticle = clone({
+    ...VideoArticle,
+    seriesArticle: SeriesArticle,
+  })
+
+  const getWrapper = (props) => {
```

Comment:
Can set a default value (`const getWrapper = (props = {})`) so as to avoid extra `{}` passing

Reply:
That was the original setup, but it raised another issue on pre-push, where `component.instance().onPlayToggle` raised a type error.  I wasn't sure how to type a function on the enzyme instance.